src/services/user.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+let users = [];
+
+const resetUsers = () => {
+  users = [];
+};
+
+const getAll = () => {
+  return users;
+};
+
+const getById = (userId) => {
+  const foundUser = users.find(user => user.id === userId);
+
+  return foundUser || null;
+};
+
+const addUser = (name) => {
```

Comment:
Add validation for the 'name' parameter to prevent adding a user with an empty or invalid name. Return 400 if the 'name' is not provided or doesn't meet the requirements.